### ISSUE TITLE
Patch or Replace react-swipeable-views

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#3325]
We use a package called react-swipeable-views for our reflection columns & timer menu tabs.
It's absolutely fantastic, but the community that uses it is playing OSS chicken on who's going to bite the bullet to refactor it to be compliant with the next version of react: https://github.com/oliviertassinari/react-swipeable-views/issues/534

I'd like to avoid being the one to bite that bullet, but it is blocking us from upgrading to the new react.
(along with #3324). This task should come immediately after #3324, since that will buy us all the time possible.

**Comment by jordanh** [comment]
Closing in favor of #1938
